bash

```diff
@@ -25,3 +25,8 @@ echo ${PIPESTATUS[0]}  # replace 0 with N
 
 # To create a lockfile:
 ( set -o noclobber; echo > my.lock ) || echo 'Failed to create lock file'
+
+# To redirect block of text with variables being substituted and other expansions performed to some command:
+<command> << DELIMITER
+  <contents>
```

Comment:
I recommend removing the indent. I understand that it improves readability, but it can be confusing to users, because the indent will become part of the contents.